eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -276,32 +270,59 @@ private void clusterSubmittedVariants(List<? extends SubmittedVariantEntity> sub
                                                               SubmittedVariantEntity.class);
         BulkOperations dbsnpBulkOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
                                                                    DbsnpSubmittedVariantEntity.class);
+        BulkOperations bulkHistoryOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
+                SubmittedVariantOperationEntity.class);
+        BulkOperations dbsnpBulkHistoryOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
+                DbsnpSubmittedVariantOperationEntity.class);
         long numUpdates = 0;
         long numDbsnpUpdates = 0;
         for (SubmittedVariantEntity submittedVariantEntity : submittedVariantEntities) {
             if (submittedVariantEntity.getClusteredVariantAccession() != null) {
                 // no need to update the rs of a submitted variant that already has the correct rs
                 continue;
             }
-            Query query = query(where("_id").is(submittedVariantEntity.getId()));
-            Update update = new Update();
-            update.set(RS_KEY, getClusteredVariantAccession(submittedVariantEntity));
+
+            long rsid = getClusteredVariantAccession(submittedVariantEntity);
+            // Query to update the RSid in submittedVariantEntity
+            Query rs_query = query(where("_id").is(submittedVariantEntity.getId()));
```

Comment:
nowadays I don't mind camelCase or underscore, but it should be as consistent as possible, especially locally. The exception I made using underscores was for very long test names.

Reply:
Right! That's the pythonista in me surfacing instinctively. I guess I'll have to actively repress that :)